File: equalizeTheArray.py
def equalizeArray(arr):
    hehe = {}
    new_arr = []
    for i in arr:
        if i not in hehe:
            hehe[i] = 1
            new_arr.append(arr.count(i))

    maximum_count = max(new_arr)
    new_arr.sort()
    new_arr.pop()

    return sum(new_arr)


arr = [1, 2, 2, 3]
print(equalizeArray(arr))

File: equalizeTheArray.js
const equalizeArray = (arr) => {
    let dict = {};
    
    for (let i=0; i<arr.length; i++) {
        if (!(arr[i] in dict)) {
            dict[arr[i]] = 1;
        } else {
            dict[arr[i]] += 1;
        }
    }

    const counts = Object.values(dict);
  
    return arr.length - Math.max(...counts);
}

arr = [1,2,2,2,1,3,3,4,5,4,5,6,7,6,7];
console.log(equalizeArray(arr));